akka-contrib/docs/receive-pipeline.rst

```diff
@@ -84,8 +99,24 @@ a simple rule.
   of the interceptors handles a message, the current Actor's behavior will receive it, and if the
   behavior doesn't handle it either, it will be treated as usual with the unhandled method.
 
-But some times it is desired for interceptors to break the chain. You can do it by explicitly ignoring
-the messages with empty cases or just not calling the inner interceptor received by parameter.
+But sometimes it is desired for interceptors to break the chain. You can do it by explicitly indicating 
+that the message has been completely handled by the interceptor by returning the 
+:class:`HandledCompletely` result.
 
 .. includecode:: @contribSrc@/src/test/scala/akka/contrib/pattern/ReceivePipelineSpec.scala#unhandled
 
+Around advice
```

Comment:
Here too; We're not using the word Advice anywhere in the APIs, so let's not add it here to avoid confusing users